src/config.rs

```diff
@@ -131,10 +131,11 @@ impl MiniserveConfig {
             ]
         };
 
-        let random_route = if args.random_route {
-            Some(nanoid::nanoid!(6, &ROUTE_ALPHABET))
-        } else {
-            None
+        let route_prefix = match (args.route_prefix, args.random_route) {
+            (Some(prefix), _) if prefix.starts_with('/') => prefix,
+            (Some(prefix), _) => format!("/{}", prefix),
```

Comment:
```suggestion
            (Some(prefix), _) => format!("/{}", prefix.trim_matches('/')),
```

This should normalize the prefix to handle the case of leading and trailing slash. In this case, the previous match arm should be removed.